.github/workflows/smoke-test-daily.yml

```diff
@@ -219,7 +220,7 @@ jobs:
               working-directory: plugins/woocommerce
               env:
                   E2E_MAX_FAILURES: 15
-              run: pnpm exec playwright test --config=tests/e2e-pw/daily.playwright.config.js
+              run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js
```

Comment:
In my previous PR, I used the `daily.playwright.config.js` file so that only the merchant tests would run. In this PR, all merchant and shopper tests should now be runnable, therefore I'm reverting it back to the default config file.